src/cookbook/index.md

```diff
@@ -1,77 +1,77 @@
+Con todas estas diferencias, por favor tome en cuenta que el libro de recetas _no es_ un manual paso a paso. Para la mayoría del contenido, se espera que tenga una comprensión básica de conceptos como HTML, CSS, Javascript, npm/yarn, etcétera.
 :::
 
-## Cookbook Contributions
+## Contribuciones del libro de recetas
 
-### What we're looking for
+### Lo que estamos buscando
 
-The Cookbook gives developers examples to work off of that both cover common or interesting use cases, and also progressively explain more complex detail. Our goal is to move beyond a simple introductory example, and demonstrate concepts that are more widely applicable, as well as some caveats to the approach.
+El libro de recetas proporciona ejemplos típicos de la vida real o interesantes para que los desarrolladores puedan usar como modelo, y también explica progresivamente detalles más complejos. Nuestro objetivo es ir más allá de un simple ejemplo introductorio, y demostrar los conceptos que son más aplicables, así como algunas advertencias al enfoque utilizado.
 
-If you're interested in contributing, please initiate collaboration by filing an issue under the tag **cookbook idea** with your concept so that we can help guide you to a successful pull request. After your idea has been approved, please follow the template below as much as possible. Some sections are required, and some are optional. Following the numerical order is strongly suggested, but not required.
+Si está interesado en contribuir, inicie la colaboración reportando un _issue_ con su idea y con la etiqueta **cookbook idea**, para que podamos guiarlo hacia la creación de un _pull request_ exitoso. Después de que su idea haya sido aprobada, por favor siga tanto como sea posible la plantilla que se muestra a continuación. Algunas secciones son requeridas, y otras son opcionales. Seguir el orden numérico se sugiere fuertemente, pero no es obligatorio.
 
-Recipes should generally:
+Las recetas generalmente deben:
 
-- Solve a specific, common problem
-- Start with the simplest possible example
-- Introduce complexities one at a time
-- Link to other docs, rather than re-explaining concepts
-- Describe the problem, rather than assuming familiarity
-- Explain the process, rather than just the end result
-- Explain the pros and cons of your strategy, including when it is and isn't appropriate
-- Mention alternative solutions, if relevant, but leave in-depth explorations to a separate recipe
+- Resolver un problema específico y común.
+- Comenzar con el ejemplo más simple posible.
+- Introducir las complejidades una a la vez.
+- Enlazar a otros documentos, en lugar de re-explicar conceptos.
+- Describir el problema, en lugar de asumir familiaridad.
+- Explicar el proceso, en lugar de solo mostrar el resultado.
+- Explicar las ventajas y desventajas de su estrategia, incluso, cuándo es apropiado y cuándo no.
```

Comment:
Should it be `apropiada`? As you are talking about `la estrategia`? 🤔

Reply:
Yes! You're right!